Description: Remove special optimization workaround for sparc.
 This was probably due to a GCC bug that most probably has been fixed.
 In any event, bugs that suddenly disappear by removing optimization
 should be exposed and fixed.  Not forwarded yet.
Author: Yavor Doganov <yavor@gnu.org>
Forwarded: no
Last-Update: 2020-04-30
---

--- gnustep-gui.orig/Source/GNUmakefile.preamble
+++ gnustep-gui/Source/GNUmakefile.preamble
@@ -65,14 +65,6 @@
 # ADDITIONAL_LIB_DIRS = 
 
 #
-# Sparc systems cannot load tiff files due to some problem compiling
-# this file with optimization. This includes Solaris [78] and sparc-*-linux-gnu
-#
-ifeq ($(findstring sparc, $(GNUSTEP_TARGET_CPU)), sparc)
-NSBitmapImageRep+JPEG.m_FILE_FILTER_OUT_FLAGS = -O%
-endif
-
-#
 # Flags dealing with installing and uninstalling
 #
 
